src/Veldrid.Tests/TextureTests.cs

```diff
@@ -935,6 +935,16 @@ public void CreateSmallTexture(PixelFormat format)
             Assert.Equal(1u, tex.Height);
         }
 
+        [Fact]
+        public void CreateTextureView2DArray()
```

Comment:
We need a test that functionally stresses the new codepaths that will be enabled by this new feature. This test doesn't really do anything except check that the constructors have assigned reasonable values to stuff. That is valuable by itself, but the hardest part of this feature is making sure everything behaves correctly and identically across backends. A test that does something like the following would give a lot more confidence:

1. Create an array texture and fill it with some color, different on each layer
2. Create a NON-array view of a specific layer
3. Sample from the texture
4. Assert that the retrieved color is correct depending on the layer

and/or

1. Create a NON-array texture and fill it with some color
2. Create a 2D-Array view of it
3. Sample from the texture view at a certain layer
4. Assert that the correct color was retrieved

Reply:
I agree, but I won't be able to do that, feel  free to close PR is it makes more breaking change.